requirements_manager/requirements_manager_class.py

```diff
@@ -0,0 +1,82 @@
+import os.path
+from typing import Dict, List, Optional
+import configparser
+from configparser import SectionProxy, ConfigParser
+
+from .errors import NoPackagesInPipfileError, PipfilePathDoesNotExistError
+from .operator_enums import OperatorEnum
+
+
+# todo - Store this as a Github gist or re-usable, cross-repo package that can be used across Monolith projects
+
+class RequirementsManager:
+    """
+    Manages reading the Pipfile and converting it into a correctly formatted list of strings for use in the
+    'install_requires' and 'extra_requires' args of the setup process.
+
+    Attributes:
+        self.config: (Dict) list of raw line by line output of Pipfile
+        self.packages: (Optional[Dict]) all packages and versions excl. dev-packages
+        self.dev_packages: (Optional[Dict]) all dev-packages
+    """
+    def __init__(self, pipfile_loc: str) -> None:
+        """
+        :param pipfile_loc: (str) location of Pipfile
+        :return: None
+        :raises: (PipfilePathDoesNotExistError) if Pipfile path does not exist
+        """
+        if os.path.exists(pipfile_loc):
```

Comment:
Minor: Maybe slightly more readable

```suggestion
    if not os.path.exists(pipfile_loc):
      raise PipfilePathDoesNotExistError()
    self.config: ConfigParser = configparser.ConfigParser()
    self.config.read(pipfile_loc)
```